### ISSUE TITLE
FileInfo_Query_FileInternalInformation could run on OTHERFS

### ISSUE THREAD

**Issue by gwr** [microsoft/WindowsProtocolTestSuites#259]
FileInfo_Query_FileInternalInformation gives up early for no obvious reason when the FS type is OTHERFS and the 64-bit ID is non-zero.

In this test, our FS actually _does_ return 64-bit File IDs.
I realize this is not a proper fix, but here's what I did for now:

```
diff --git a/TestSuites/FileServer/src/FSA/TestSuite/FileInformation/FileInfo_Query_FileInternalInformation.cs b/TestSuites/FileServer/src/FSA/TestSuite/FileInformation/FileInfo_Query_FileInternalInformation.cs
index eab99cbf..1ce746fb 100644
--- a/TestSuites/FileServer/src/FSA/TestSuite/FileInformation/FileInfo_Query_FileInternalInformation.cs
+++ b/TestSuites/FileServer/src/FSA/TestSuite/FileInformation/FileInfo_Query_FileInternalInformation.cs
@@ -89,7 +89,8 @@ namespace Microsoft.Protocols.TestSuites.FileSharing.FSA.TestSuite
             //See [MS-FSCC] section <10> for the supportted file system list.
             if (this.fsaAdapter.FileSystem == FileSystem.CDFS
                 || this.fsaAdapter.FileSystem == FileSystem.CSVFS
-                || this.fsaAdapter.FileSystem == FileSystem.OTHERFS)
+//XXX           || this.fsaAdapter.FileSystem == FileSystem.OTHERFS
+)
             {
                 this.TestSite.Assert.AreEqual(0, fileInternalInformation.IndexNumber, string.Format("The current file system is {0}. [MS-FSCC] section 2.1.9: For file systems that do not support a 64-bit file ID, this field MUST be set to 0, and MUST be ignored. ", this.fsaAdapter.FileSystem));
             }
```

**Comment by Dingshouqing** [comment]
Thanks for reporting this issue. We’ll update our code to check if the current file system supports 64-bit file ID by below property value and then assert if the value is correct or not.
![1](https://user-images.githubusercontent.com/17266569/150067418-f1904ab4-2256-4a54-bff6-030d7aa24241.png)

We will keep you updated once it is done.

Best Regards,
Windows Protocol Test Suites Development Team

**Comment by Dingshouqing** [comment]
The fix is available on main branch by commit [e9797062c5d0e277a4bf1e9745bff79e59c59373](https://github.com/microsoft/WindowsProtocolTestSuites/commit/e9797062c5d0e277a4bf1e9745bff79e59c59373).
You could clone our latest code, build a new package, set Is64bitFileIdSupported to “true”, and run tests again.
 
![3](https://user-images.githubusercontent.com/17266569/151508482-0202712c-b31e-4ba2-a393-c593a54dbe5f.png)

Best Regards,
Windows Protocol Test Suites Development Team

**Comment by Dingshouqing** [comment]
Hi Gordon,

Since this thread has been inactive for some time, we will close it for now. Please feel free to contact us if there is any further question.

Best Regards,
Windows Protocol Test Suites Development Team
